pulsar-client/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -52,6 +51,14 @@
      */
     ConsumerBuilder<T> clone();
 
+    /**
+     * Load the configuration from provided <tt>config</tt> map.
```

Comment:
We could have a small example of parameters that can be configured through the map here.